src/gui/folderstatusmodel.cpp

```diff
@@ -1200,7 +1197,7 @@ void FolderStatusModel::slotNewBigFolder()
         return;
     }
 
-    _folders[folderIndex].resetSubs(this, index(folderIndex));
```

Comment:
i still believe the reset is required.

Reply:
As said, I think its covered by https://github.com/owncloud/client/blob/58d54d68e09e5eca9ef83bf9173fe2ac79732a44/src/gui/folderstatusmodel.cpp#L564